vendor.conf

```diff
@@ -46,23 +46,23 @@ go.etcd.io/bbolt                                    a0458a2b35708eef59eb5f620ceb
 go.opencensus.io                                    9c377598961b706d1542bd2d84d538b5094d596e # v0.22.0
 golang.org/x/net                                    f3200d17e092c607f615320ecaad13d87ad9a2b3
 golang.org/x/sync                                   42b317875d0fa942474b76e1b46a6060d720ae6e
-golang.org/x/sys                                    c990c680b611ac1aeb7d8f2af94a825f98d69720 https://github.com/golang/sys
+golang.org/x/sys                                    52ab431487773bc9dd1b0766228b1cf3944126bf
```

Comment:
nit: This seems on Nov 16, 2019. Can we vendor the latest commit?

Reply:
picked this commit to align it again with containerd/cri. want me to update as part of this PR, or a follow-up?

Reply:
can be follow-up